### PR TITLE
Refactor rosco to run jobs directly instead of relying on rush.

### PR DESCRIPTION
**Pull request by duftler**
Drop dependency on cassandra.

### CODE REVIEW

**Comment by duftler** [comment]
@lwander or @ttomsu please review.
@gardleopard please review.

**Comment by duftler** [comment]
The refactored rosco will respect pre-existing bakes in the same redis cluster (including in-flight bakes).

**Comment by gardleopard** [comment]
Great @duftler! I have tested this in our dev environment and it works. Parallel building works and it does not interfere with old version of rosco/rush running bakes with the same redis backend.

Making the rush dependency and at the same time removing 257 lines of code is awesome! 
screen shot 2016-06-15 at 22 28 06 👍

File: build.gradle [hunk 1]
```diff
@@ -20,7 +20,7 @@ allprojects {
   group = "com.netflix.spinnaker.rosco"
 
   spinnaker {
-    dependenciesVersion = "0.19.0"
+    dependenciesVersion = "0.40.0"
```
Comment (ttomsu) [inline build.gradle]: Yikes - that's a jump!

**Comment by ttomsu** [comment]
LGTM

**Pull request merged**